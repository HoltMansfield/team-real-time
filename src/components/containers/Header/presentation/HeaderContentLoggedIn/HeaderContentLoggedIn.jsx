import React from 'react'
import { Link } from 'react-router-dom'
import AppBar from 'material-ui/AppBar'


export default function HeaderContentLoggedIn(props) {
  return (
    <AppBar
      title="team-real-time"
      iconClassNameRight="muidocs-icon-navigation-expand-more"
    />
  )
}
