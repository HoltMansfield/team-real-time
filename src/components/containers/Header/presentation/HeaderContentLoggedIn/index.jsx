import React from 'react'
import { Link } from 'react-router-dom'
import ShowWhen from '../../../../presentation/ShowWhen'


export default function HeaderContentLoggedIn(props) {
  return (
    <div>
      header
    </div>
  )
}
