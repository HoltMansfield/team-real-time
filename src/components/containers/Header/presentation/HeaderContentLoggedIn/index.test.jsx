import React from 'react'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'
import HeaderContentLoggedIn from '../HeaderContentLoggedIn'

it('renders correctly', () => {
  const props = {
    loggedInUser: {}
  }
  const componentTree = (
    <BrowserRouter>
      <HeaderContentLoggedIn {...props} />
    </BrowserRouter>
  )

  const component = renderer
    .create(componentTree)
    .toJSON()
  expect(component).toMatchSnapshot()
})
