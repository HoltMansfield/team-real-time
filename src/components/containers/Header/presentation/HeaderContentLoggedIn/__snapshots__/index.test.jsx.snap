// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="menu-container"
>
  <div
    className="menu-tmx"
  />
  <div
    className="row menu-tmx-content"
  >
    <div
      className="columns medium-4 large-4"
    >
      <img
        alt="TMX Matrix"
        className="tmxlogo-header"
        src="tmx_matrix_logo_small.png"
        title="TMX Matrix"
      />
    </div>
    <div
      className="columns medium-8 large-8"
    >
      <ul
        className="tmx-header-list"
      >
        <li>
          <a
            href="/dashboard"
            onClick={[Function]}
          >
            Dashboard
          </a>
        </li>
        <li>
          <a
            href="/watchlist"
            onClick={[Function]}
          >
            Watchlist
          </a>
        </li>
        <li>
          <a
            href="/discover"
            onClick={[Function]}
          >
            Discover
          </a>
        </li>
        <li>
          <a
            href="/messages"
            onClick={[Function]}
          >
            Messages
          </a>
        </li>
        <li
          className="loggedInSearch"
        >
          <span
            className="icon-search"
          />
        </li>
        <li
          className="loggedInUser"
        >
          <a
            href="/dashboard"
            onClick={[Function]}
          >
            <img
              alt="User"
              src="profile_generic.png"
              title="User"
            />
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
